playbooks/set-gpu-clocks.yml

```diff
@@ -0,0 +1,16 @@
+---
+# set gpu clocks on all worker nodes
+- hosts: compute-node
```

Comment:
Can we instead use `slurm-node:kube-node` instead of creating another inventory group? I think the inventory file is getting needlessly long which makes it more difficult to use.

Reply:
I've encountered a few cases where people are trying to manage their GPU clusters through deepops _without_  Slurm or K8S. If this is just a use case we won't support or recommend user customization I can make this change.

Reply:
In that case just use `all`. There’s no reason to create another host group unless you need to differentiate between different sets of nodes (like in a client/server case).